### PR TITLE
Firefox 105 supports api.OffscreenCanvas.getContext.webgpu_context

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Firefox and Safari for the `getContext.webgpu_context` member of the `OffscreenCanvas` API. The data comes from the [mdn-bcd-collector](https://mdn-bcd-collector.gooborg.com) project (v10.11.0).

_Check out the [collector's guide on how to review this PR](https://github.com/openwebdocs/mdn-bcd-collector#reviewing-bcd-changes)._

Tests Used: https://mdn-bcd-collector.gooborg.com/tests/api/OffscreenCanvas/getContext/webgpu_context

### CODE REVIEW

**Review by Elchi3 — COMMENTED** [review]
Looking at the api/GPU*.json files, I'm skeptical about this change. I don't think it has shipped in Firefox/Safari stable yet.

**Comment by queengooborg** [comment]
Screenshot 2024-07-23 at 00 36 46

Hmm, in Safari, it seems like it is actually supported in worker contexts...?  Possibly a case of early exposure?

**Comment by Elchi3** [comment]
Possibly accidental early expose on workers, yes. cc @jdatapple

File: api/OffscreenCanvas.json [hunk 1]
```diff
@@ -395,7 +395,7 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "preview",
+                "version_added": "105",
```
Comment (caugner) [inline api/OffscreenCanvas.json:398]: This is confirmed by https://bugzilla.mozilla.org/show_bug.cgi?id=1779009.

File: api/OffscreenCanvas.json [hunk 2]
```diff
@@ -409,14 +409,14 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "preview"
+                "version_added": "17.4"
```
Comment (caugner) [inline api/OffscreenCanvas.json:412]: @jdatapple Can you confirm whether `webgpu_context` is intentionally exposed in worker contexts?

Comment (caugner) [reply api/OffscreenCanvas.json:412]: @queengooborg As long as we have on clarity, I would suggest to play safe, revert this bit, and stash this outstanding question to an issue.

```suggestion
                "version_added": "preview"
```

**Pull request merged**